terraform/environments/mlra/modules/cloudwatch-alarms/variables.tf

```diff
@@ -0,0 +1,124 @@
+variable "alarmname" {
+  type    = string
+  default = ""
+}
+
+variable "oper" {
```

Comment:
The module will expect these variables to be satisfied, so there are quite a few more inputs required  here than I think is expected

Reply:
variables removed